### PR TITLE
addding s3 bench Di combinations test_motr_S3bench_DI_tests.py

### PR DESCRIPTION
**Pull request by venkateshvb**
# Problem Statement
S3bench DI 41 combinations

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: comptests/motr/test_motr_S3bench_DI_tests.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
...7, "noOfClients":10, "noOfSamples":10, "objectSize":"9500b"}, {"TESTCASE_NO" : 28, "noOfClients":10, "noOfSamples":10, "objectSize":"17000b"}, {"TESTCASE_NO" : 29, "noOfClients":10, "noOfSamples":10, "objectSize":"19000b"}, {"TESTCASE_NO" : 30, "noOfClients":10, "noOfSamples":10, "objectSize":"19500b"}, {"TESTCASE_NO" : 31, "noOfClients":10, "noOfSamples":10, "objectSize":"50000b"}, {"TESTCASE_NO" : 32, "noOfClients":10, "noOfSamples":10, "objectSize":"70000b"}, {"TESTCASE_NO" : 33, "noOfClients":10, "noOfSamples":10, "objectSize":"75000b"}, {"TESTCASE_NO" : 34, "noOfClients":10, "noOfSamples":10, "objectSize":"90000b"}, {"TESTCASE_NO" : 35, "noOfClients":10, "noOfSamples":10, "objectSize":"95000b"}, {"TESTCASE_NO" : 36, "noOfClients":10, "noOfSamples":10, "objectSize":"123123b"}, {"TESTCASE_NO" : 37, "noOfClients":10, "noOfSamples":10, "objectSize":"123123123b"}, {"TESTCASE_NO" : 38, "noOfClients":10, "noOfSamples":10, "objectSize":"2312b"}, {"TESTCASE_NO" : 39, "noOfClients":10, "noOfSamples":10, "objectSize":"6756b"}, {"TESTCASE_NO" : 40, "noOfClients":10, "noOfSamples":10, "objectSize":"34123b"}, {"TESTCASE_NO" : 41, "noOfClients":10, "noOfSamples":10, "objectSize":"2134123b"}]
```
Comment (zohebkhann) [inline comptests/motr/test_motr_S3bench_DI_tests.py:11]: Use a yaml file to store these configs i.e samples, clients, object sizes.

File: comptests/motr/test_motr_S3bench_DI_tests.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+     return list_s3bench_DI_comb
+
+def main():
+    acsKey    = args.a
+    secretKey = args.s
+    endPoint  = args.e
+    for test in s3bench_DI_combinations():
+        command = "s3bench -region us-east-1 -accessKey {} -accessSecret {} -bucket test1 -endpoint {} -numClients {} -numSamples {} -objectSize {}".format(acsKey, secretKey, endPoint, test['noOfClients'], test['noOfSamples'], test['objectSize'])
+        os.system(command)
```
Comment (zohebkhann) [inline comptests/motr/test_motr_S3bench_DI_tests.py:20]: There are no checks if the tests are failing or passing?.

File: comptests/motr/test_motr_S3bench_DI_tests.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+if __name__ == '__main__':
+    main()
```
Comment (zohebkhann) [inline comptests/motr/test_motr_S3bench_DI_tests.py:23]: Looks like we are planning to run it like a script? Why are we adding this to the test framework? Are there any related TCs in the Test JIRA? I don't see any test IDs.

File: comptests/motr/test_motr_S3bench_DI_tests.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+     return list_s3bench_DI_comb
+
+def main():
+    acsKey    = args.a
+    secretKey = args.s
+    endPoint  = args.e
+    for test in s3bench_DI_combinations():
+        command = "s3bench -region us-east-1 -accessKey {} -accessSecret {} -bucket test1 -endpoint {} -numClients {} -numSamples {} -objectSize {}".format(acsKey, secretKey, endPoint, test['noOfClients'], test['noOfSamples'], test['objectSize'])
```
Comment (zohebkhann) [inline comptests/motr/test_motr_S3bench_DI_tests.py:19]: We are just running s3bench here in this loop, what else are we doing to test the DI feature here?

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete

**Pull request closed without merging**